madgiraffestudios/src/components/Navigation/NavigationBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import giraffeicon from '../../Images/icon.png';
+
+
+const NavigationBar = () => (
+
+<div className="container">
+    <Navbar expand="lg">
+        <Navbar.Brand href="/home"><img src={giraffeicon} height="50px" width="50px" /></Navbar.Brand>
```

Comment:
consider using reactstrap here instead - also, adding ml and mr will increase css formatting for mobile devices/view.  

- [ ] Consider using spacing utilities  more like mx/px values instead

Reply:
addressed